python/python/glide/async_commands/core.py

```diff
@@ -294,6 +294,25 @@ async def strlen(self, key: str) -> int:
         """
         return cast(int, await self._execute_command(RequestType.Strlen, [key]))
 
+    async def rename(self, key: str, new_key: str) -> TOK:
+        """
+        Renames `key` to `new_key`.
+        If `newkey` already exists it is overwritten.
+        In Cluster mode, both `key` and `newkey` must be in the same hash slot,
+        meaning that in practice only keys that have the same hash tag can be reliably renamed in cluster.
+        See https://redis.io/commands/rename/ for more details.
+
+        Args:
+            key (str) : The key to rename.
+            new_key (str) : The new name of the key.
+
+        Returns:
+            OK: If the `key` was successfully renamed, return "OK". If `key` does not exist, an error is thrown.
```

Comment:
```suggestion
            OK: If the `key` was successfully renamed. If `key` does not exist, an error is thrown.
```

Some docs have OK, some - TOK. Consider fixing this in another PR

Reply:
I'm not sure what the suggested change is trying to accomplish here. "If the `key` was successfully renamed." is not a complete sentence.